src/frame_graph/frame_graph.hpp

```diff
@@ -685,6 +703,10 @@ namespace wr
 		const std::uint64_t m_uid;
 		static std::uint64_t m_largest_uid;
 		static std::stack<std::uint64_t> m_free_uids;
+
+		/*! Store old resolution for proper rendertarget scaling. */
+		uint32_t m_old_window_size_width = 1240;
```

Comment:
We do std::uint32_t to be consistent. uint32_t is defined in `stdint.h` and std::uint32_t in `cstdint`